app/controllers/users_controller.rb

```diff
@@ -0,0 +1,65 @@
+class UsersController < ApplicationController
+  def index
+    @user_favorites = TunesTakeoutWrapper.favorites(current_user.uid)
+    @favorites = []
+    @user_favorites["suggestions"].each do |user_favorite|
+      @favorites << TunesTakeoutWrapper.find_suggest(user_favorite)["suggestion"]
+    end
+    @your_favorites = []
+    @favorites.each do |favorite|
+      fave_sugg = {}
+      yelp = Food.yelp_find(favorite["food_id"])
+      spotify = Music.spotify_find(favorite["music_type"], favorite["music_id"])
+      fave_sugg[:suggestion_id] = favorite["id"]
+      fave_sugg[:business_name] = yelp.business.name
+      fave_sugg[:business_url] = yelp.business.url
+      fave_sugg[:business_image] = yelp.business.image_url
+      fave_sugg[:business_rating_image] = yelp.business.rating_img_url
+      fave_sugg[:business_phone] = yelp.business.phone
+      fave_sugg[:music_name] = spotify.name
+      fave_sugg[:music_type] = spotify.type
+      fave_sugg[:music_embed] = spotify.uri
+      fave_sugg[:music_url] = spotify.external_urls["spotify"]
```

Comment:
I'm not sure why you have this logic in this controller. Put it in the model if you can, or a helper method if you can't.